feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -285,7 +275,79 @@ class BillingWrapper(
         )
     }
 
-    fun consumePurchase(
+    override fun consumeAndSave(
+        shouldTryToConsume: Boolean,
+        purchase: PurchaseWrapper
+    ) {
+        if (purchase !is GooglePurchaseWrapper) return
```

Comment:
similar question here as the amazon version, should this crash?

Reply:
like, it'd only happen in a bad config case, right?

Reply:
I answered in the other question, but just in case, this is only for smartcasting. It should never happen. We can crash tho, I am fine with that.